test/test_comment.py

```diff
@@ -20,14 +20,16 @@
 import sys
 
 sys.path[0:0] = [""]
-
+from asyncio import iscoroutinefunction
 from test import IntegrationTest, client_context, unittest
 from test.utils import EventListener
 
 from bson.dbref import DBRef
 from pymongo.operations import IndexModel
 from pymongo.synchronous.command_cursor import CommandCursor
 
+_IS_SYNC = True
+
 
 class Empty:
```

Comment:
This Empty class looks very suspicious. What's the point of it?

Reply:
Uhhh honestly not sure. It was added in [PYTHON-2682](https://jira.mongodb.org/browse/PYTHON-2682).

Reply:
cleaned up `_test_ops` and now `Empty` isn't needed anymore :)